src/main/java/org/broadinstitute/hellbender/tools/reference/CompareReferences.java

```diff
@@ -204,6 +256,193 @@ private void displayBySequenceName(Set<ReferenceSequenceTable.TableRow> rows, St
         System.out.println();
     }
 
+    private void runFullAlignment(ReferencePair refPair, ReferenceSequenceTable table){
+        List<File> snpsFiles = new ArrayList<>();
+        if(refPair.getAnalysis().contains(ReferencePair.Status.DIFFER_IN_SEQUENCE)) {
+            // find the mismatch sequence
+            for (String sequenceName : table.getAllSequenceNames()) {
+                Set<ReferenceSequenceTable.TableRow> rows = table.queryBySequenceName(sequenceName);
+                if (rows.size() == 2) {
```

Comment:
Add a one-line comment here explaining this line -- eg., "two rows means two differing sequences for the same name"